libs/designsystem/data-table/src/sortable/sortable.component.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { MockComponent } from 'ng-mocks';
+import { createHostFactory, Spectator } from '@ngneat/spectator';
+import { IconComponent } from '@kirbydesign/designsystem/icon';
+import { DesignTokenHelper } from '@kirbydesign/designsystem/helpers';
+import { TableSortableComponent } from './sortable.component';
+
+const getFontWeight = DesignTokenHelper.fontWeight;
+
+describe('TableSortableComponent', () => {
+  let spectator: Spectator<TableSortableComponent>;
+
+  const createHost = createHostFactory({
+    component: TableSortableComponent,
+    declarations: [MockComponent(IconComponent)],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(`
+    <table class="kirby-table">
+      <th [sortable]="true">Data1</th>
+    </table>
+    `);
+  });
+
+  describe('by default', () => {
+    it('should create', () => {
+      expect(spectator.component).toBeTruthy();
+    });
+
+    it('should hide the sorting icon', () => {
+      const icon = spectator.query('kirby-icon');
+
+      expect(icon).toHaveComputedStyle({
+        visibility: 'hidden',
+      });
```

Comment:
Consider just using built in test methods:
```suggestion
      expect(icon).toBeHidden();
```

Reply:
@jakobe I agree! :) But the test fails when I try to use `toBeHidden()`. Do you know if `toBeHidden()` works with `visibility: hidden` and not only `display: none`? I tried to look for it in the Jasmine docs, but cannot find anything.

Reply:
Hmm dunno 🤷🏻 
Leave it then ;)